### PR TITLE
Works Section of Portfolio Project (Mobile Version)

### PR DESCRIPTION
**Pull request by engrabdullahriaz**
1. Added all the requirements for colors, fonts, and box-model from the Figma design
2. No linters error
3. README is descriptive
4. created a new branch

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons-list">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist"></i></a></li>
+      </ul>
```
Comment (vic778) [inline index.html:54]: kindly the social media should be flexed. You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">HTML</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">CSS</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">Javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
```
Comment (vic778) [inline index.html:109]: kindly the `h3` enter in the `see project` fix that issue

File: index.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                hestiate to contact me.
+              </p>
+            </div>
+            <div>
+              <button type="button" class="about_btn">Get My Resume</button>
+            </div>
+          </div>
+          <div class="about-language">
+            <div class="lang-title">
+              <h2>Languages</h2>
```
Comment (vic778) [inline index.html]: Kindly this title miss an arrow down icon and try to reduce the title is too big differently to the Figma template

File: index.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
+              </ul>
+            </div>
+            <div class="image-language">
+              <img src="images/htmlcssjs.png" alt="web images">
+            </div>
+            <div class="skills-language">
+              <h3>Framework</h3>
+              <ul>
+                <li class="arrow-right"><i class="fa fa-arrow-right"></i></li>
```
Comment (vic778) [inline index.html:199]: Kindly this font-awesome is not working according to the Figma template

File: index.html [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+                </ul>
+            </div>
+              <hr>
+              <div class="skills-info">
+              <h4 class="skills-title">skills</h4>
+              <ul>  
+              <li class="arrow-side"><i class="fa fa-arrow-right"></i></li>
```
Comment (vic778) [inline index.html:206]: Kindly this font-awesome is not working according to the Figma template

File: index.html (hunk 2 above)
Comment (vic778) [inline index.html]: Kindly this section should display the image of each projects. and the project should have the see project button according to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @engrabdullahriaz 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
-  Good video description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (engrabdullahriaz) [reply index.html:107]: <div class="lang-title">
              <h2>Languages</h2>
              <ul>
              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
              </ul>
            </div>

Comment (engrabdullahriaz) [reply index.html]: <h2>Languages</h2>
              <ul>
              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
              </ul>

File: index.html [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <h3><button class="btn-card">See project</button></h3>
```
Comment (AnselemOdims) [inline index.html]: - [ ] As pointed out by the previous reviewer, please make sure that each card has an image like the original design.
- [ ] Also, please ensure to include the number of cards in the original design

### Your Design
image

### Original Design
image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+            </div>
+            <div>
+              <button type="button" class="about_btn">Get My Resume</button>
+            </div>
+          </div>
+          <div class="about-language">
+            <div class="lang-title">
+              <h3>Languages</h3>
+              <ul>
+              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please kindly ensure the feedback by the previous reviewer are implemented and the about section looks like the original design

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @engrabdullahriaz , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Just One More Thing 🔸

## Hi @engrabdullahriaz 👋

Great work on making the changes requested by a previous reviewer 👏.

You've done well implementing some of the requested changes 💯, but **there are still some which aren't addressed yet**.

Though, There are some **few** issues 🐛 that you still need to work 🔧 on to go to the next project ⏭️ but you are almost there! 🏃

## Don't Feel frustrated 🤔

I am sure you worked hard 🛠️ to make these changes and actually you have to be proud of them 🥇.

And I understand how much you wanted to get this project approved ✔️ (I was at your place one day 😉). But, don't worry, You still can do it! 💪.

Just apply the changes ♻️ down below 👇 and this time it is going to be approved ✅. Just make sure that you **don't miss any of the changes** requested or introduce any **new bugs** 🐛.

> **Note:** A good tip is to check the box next to every change after you implement it ☑️. And you can also check the full website to see if it is still working after you have implemented the changes.

*Please, reach to [me](#thank-you-for-your-time-🙏-and-call-me-if-you-need-help-or-clarification-☎️) for any help 🪝 or any question ❓. I am always happy to help you 😃.*

## Required Changes ♻️

- [x] [REQUIRED] Please, implement the previous reviewer's requested changes (you didn't implement any)

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+      <!--end about me section-->
+     <section class="about_me">
+        <div class="about_container">
+          <h3 class="about_me_title"> About <br>Myself</h3>
+          <div class="about_info">
+            <h3 class="about_me_title-desktop">About me</h3>
+            <div class="about_description">
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly make sure the design in conforming the Figma mockup
- [ ] Kindly separate the see project button and the about section
Screen Shot 2022-04-27 at 13 30 47
- [ ] Kindly add at least 4 projects

File: style.css [hunk 9]
```diff
@@ -0,0 +1,415 @@
...
+  .icons-list {
+    margin-left: 8rem;
+  }
+}
+
+.container-work {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto fit);
+  place-content: left;
```
Comment (aimemalaika) [inline style.css]: - [x] Kindly fix these two lines as they are invalid property values

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @engrabdullahriaz, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <button class="btn-card">See project</button>
+        </div>
+      </section>
+
```
Comment (roshan-bajgain) [inline index.html:85]: - [x] I would suggest you make changes and stick as much as possible to Figma design.
- [x] kindly make the `li` tag looks similar to the Figma design and makes the less spacing between the `h2` tag and the `li` tag.
- [x] Kindly change the styling of the `li` tag (HTML, CSS, javascript) and make sure it looks like Figma design
- [x] kindly implement all the changes in all the section
### your design
your-design
## Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: 🍾 
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by aimemalaika — APPROVED** [review]
Hi @engrabdullahriaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**